services/src/api/handlers/datasets.rs

```diff
@@ -3036,9 +2838,17 @@ mod tests {
     }
 
     #[ge_context::test]
-    async fn it_lists_layers(app_ctx: PostgresContext<NoTls>) {
-        let ctx = app_ctx.default_session_context().await.unwrap();
-        let session_id = ctx.session().id();
+    async fn it_lists_layers(app_ctx: ProPostgresContext<NoTls>) {
+        {
+            // TODO: better way to get to the root of the project
+            let mut working_dir = std::env::current_dir().unwrap();
+            if working_dir.ends_with("services") {
+                working_dir.pop();
+            }
+            std::env::set_current_dir(working_dir).unwrap();
```

Comment:
do you not have to undo this before the test exits?

Reply:
[d26f056](https://github.com/geo-engine/geoengine/pull/1006/commits/d26f056f613fc929965824d8218aa3969d58d44a)